lib/Wikia/src/Vignette/UrlGenerator.php

```diff
@@ -31,6 +31,10 @@ class UrlGenerator {
 
 	const REVISION_LATEST = 'latest';
 
+	const BASE_URL_PROD = 'http://vignette<SHARD>.wikia.nocookie.net';
```

Comment:
The spirit of keeping these out of `/lib/Wikia` is the hopes that these vignette libraries could be decoupled from MW and opened as separate libraries. If we embed Wikia specific config we can't do that or we'll have to unwind it later.